009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,433 @@
+\subsubsection{Significance of Treaties}
+\label{sec: Simulations and Analysis}
+\Cref{fig: Cumulative Deaths without Treaties} and \Cref{fig: Cumulative Deaths with Treaties} illustrate the results from simulations with treaties disabled and enabled respectively. All other factors are left as default.
+
+\begin{figure}[h!]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, WO Treaties, T7Only, 2000days, 20food, 93deaths.png}
+    \end{center}
+    \caption{Cumulative deaths without treaties (93 deaths).}
+    \label{fig: Cumulative Deaths without Treaties}
+\end{figure}
+
+\begin{figure}[h!]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, With Treaties, T7Only, 2000days, 20food, 67deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with treaties (67 deaths).}
+    \label{fig: Cumulative Deaths with Treaties}
+\end{figure}
+
+This demonstrates that the agents perform better collectively when treaties are activated. This is expected as treaties enable agents to cooperate and organize the distribution of scarce food. 
+
+\newpage
+\subsubsection{Impact of Personalities}
+\label{subsubsec: Impact of Personalities}
+
+\subsubsection{Openness}
+\label{subsubsec: Openness}
+
+\Cref{fig: High Openness} and \Cref{fig: Low Openness} detail how our agents adapt to situations/scenarios differently based on varying levels of openness. In \Cref{fig: High Openness}, all the agents are spawned with openness values greater than 70 and in \Cref{fig: Low Openness}, the agents are spawned with openness values less than 30. All other personality traits are distributed as in the default case.
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, With Treaties, T7Only, 2000days, 20food, High Openness, 434deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with high openness (434 deaths).}
+    \label{fig: High Openness}
+\end{figure}
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, With Treaties, T7Only, 2000days, 20food, Low Openness, 29deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with low openness (29 deaths).}
+    \label{fig: Low Openness}
+\end{figure}
+
+It can be observed that the agents collectively perform significantly better when openness is low, as a stabilising of the cumulative deaths curve can be seen for the agent with low openness. This can be explained by the fact that an agent with high openness will have a lower standard for accepting message requests and treaties - meaning they are more likely to agree to a request that is detrimental to their survival. Furthermore, an agent with a low openness value, will be less welcoming to a change in floor and will therefore behave more cautiously. This will also improve the agents chances of survival. 
```

Comment:
```suggestion
It can be observed that the agents collectively perform significantly better when openness is low, as a stabilising of the cumulative deaths curve can be seen for the agent with low openness. This can be explained by the fact that an agent with high openness will have a lower standard for accepting message requests and treaties - meaning they are more likely to agree to a request that is detrimental to their survival. Furthermore, an agent with a low openness value will be less welcoming to a change in floor and will therefore behave more cautiously. This will also improve the agents chances of survival. 
```